Vector.h

```diff
@@ -322,10 +322,10 @@ namespace NCL::Maths {
         template <typename T, uint32_t n>
         constexpr VectorTemplate<T, n>		Clamp(VectorTemplate<T, n>& input, VectorTemplate<T, n>& mins, VectorTemplate<T, n>& maxs) {
             VectorTemplate<T, n> output;
-            for (int i = 1; i < n; ++i) {
+            for (int i = 0; i < n; ++i) {
```

Comment:
This one IS correct though - I'll have copy pasted the previous for loops without thinking of the additional context.

Reply:
Got it, I've reverted the other changes